cli/src/commands/runTests.js

```diff
@@ -318,6 +318,13 @@ async function writeFlowConfig(testDirPath, libDefPath, includeWarnings) {
     path.join(testDirPath, '..', '..', 'node_modules'),
   ].join('\n');
   await fs.writeFile(destFlowConfigPath, flowConfigData);
+
+  const tddFilePath = path.join('src', 'lib', 'tdd_framework.js');
+  const tddFrameworkFile = (await fs.readFile(tddFilePath)).toString();
```

Comment:
I think `const tddFrameworkFile = await fs.readFile(tddFilePath, 'utf-8');` is enough here